### ISSUE TITLE
Develop pre-publication checklists for authors & network managers

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/ideas#459]
As an author, I want to ensure that my book is maximally useful to others and that it's truly ready for public consumption (i.e. I've fulfilled all relevant licensing/legal requirements). The same thing is true if I'm a network manager running a network. I would love to have a short pre-publication checklist or set of nudges that help me ensure that I've done some basic pre-publication tasks and my book is 'ready' to be published and widely distributed/consumed by discovery services.

**Comment by paradisojr** [comment]
Love this idea!

**Comment by SteelWagstaff** [comment]
Still extremely relevant, though I think it needs quite a bit of speccing out. Happy to work with @MichelleWeremczuk or others to define what this might look like?

**Comment by cmurtagh** [comment]
Yeah, 'maximally useful' definitely sounds like a CX task (and too vague, tbh).

'all relevant licensing/legal requirements' sounds like a legal nightmare - we certainly can't have this look at all like we're giving legal advice or saying doing a specific task will meet legal requirements. This can vary widely depending on jurisdiction, etc. 

I suspect the best way to look at this would be to start with a link that goes to a chapter in the guidebook that includes some common best practices.

**Comment by MichelleWeremczuk** [comment]
Agreed, this would be worthwhile. We could use methods like a guide chapter, interactive PDF, or a cascade of nudges/wizard, but there are many details to consider. We could start with a low-tech version to refine the process before implementing it in the editor.

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 3y ago with no updates in the last 8m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
